src/from1DNMRVariables.ts

```diff
@@ -17,13 +19,17 @@ type NtuplesData<DataType extends DoubleArray = DoubleArray> = Record<
   'x' | 'r' | 'i',
   MeasurementVariable<DataType>
 >;
+type RealData<DataType extends DoubleArray = DoubleArray> = Record<
+  'x' | 'r',
+  MeasurementVariable<DataType>
+>;
 type PeakData<DataType extends DoubleArray = DoubleArray> = Record<
   'x' | 'y',
   MeasurementVariable<DataType>
 >;
 
 const peakDataKeys = ['x', 'y'] as Array<keyof PeakData>;
-const ntuplesKeys = ['x', 'r', 'i'] as Array<keyof NtuplesData>;
+const ntuplesKeys = ['r', 'i'] as Array<keyof NtuplesData>;
 
 function isPeakData(
```

Comment:
It should not be called PeakData but XYData (please change everywhere). PeakData is used for mass spectrometry fragmentation but for NMR we have equidistant x values.